eng/pipelines/templates/jobs/ci.yml

```diff
@@ -52,6 +52,11 @@ jobs:
       Codeql.SkipTaskAutoInjection: false
 
     steps:
+      - template: /eng/common/pipelines/templates/steps/sparse-checkout.yml
```

Comment:
Didn't you say this break sourcelink?

Reply:
Correct.   I'm closing the PR.  We can't use sparese checkouts with source link until they release a package with this fix:
https://github.com/dotnet/sourcelink/pull/772 for this issue:

Reply:
Sparse checkout support was added to Source Link in version 8.0.0.  Re-opening this PR to bump sourcelink version.